src/shipit_static_analysis/shipit_static_analysis/workflow.py

```diff
@@ -286,3 +308,35 @@ def build_improvement_patch(self, revision, issues):
             diff_name=diff_name,
         )
         logger.info('Diff available online', url=revision.diff_url)
+
+    def index(self, revision, **kwargs):
+        '''
+        Index current task on Taskcluster index
+        '''
+        assert isinstance(revision, Revision)
+
+        if not self.on_taskcluster:
+            logger.info('Skipping taskcluster indexation', rev=str(revision), **kwargs)
```

Comment:
Nit: it should be `indexing` instead of `indexation`

Reply:
Here and in other places

Reply:
:fr: FTW